Libraries/LibGfx/PainterSkia.cpp

```diff
@@ -108,6 +111,112 @@ static SkPaint to_skia_paint(Gfx::PaintStyle const& style, ReadonlySpan<Gfx::Fil
+    }
+    if (current_compositing_and_blending_operator == "hard-light"sv) {
+        return SkBlender::Mode(SkBlendMode::kHardLight);
+    }
+    if (current_compositing_and_blending_operator == "soft-light"sv) {
+        return SkBlender::Mode(SkBlendMode::kSoftLight);
+    }
+    if (current_compositing_and_blending_operator == "difference"sv) {
+        return SkBlender::Mode(SkBlendMode::kDifference);
+    }
+    if (current_compositing_and_blending_operator == "exclusion"sv) {
+        return SkBlender::Mode(SkBlendMode::kExclusion);
+    }
+    if (current_compositing_and_blending_operator == "hue"sv) {
+        return SkBlender::Mode(SkBlendMode::kHue);
+    }
+    if (current_compositing_and_blending_operator == "saturation"sv) {
+        return SkBlender::Mode(SkBlendMode::kSaturation);
+    }
+    if (current_compositing_and_blending_operator == "color"sv) {
+        return SkBlender::Mode(SkBlendMode::kColor);
+    }
+    if (current_compositing_and_blending_operator == "luminosity"sv) {
+        return SkBlender::Mode(SkBlendMode::kLuminosity);
+    }
+    if (current_compositing_and_blending_operator == "clear"sv) {
+        return SkBlender::Mode(SkBlendMode::kClear);
+    }
+    if (current_compositing_and_blending_operator == "copy"sv) {
+        return SkBlender::Mode(SkBlendMode::kSrc);
+    }
+    if (current_compositing_and_blending_operator == "source-over"sv) {
+        return SkBlender::Mode(SkBlendMode::kSrcOver);
+    }
+    if (current_compositing_and_blending_operator == "destination-over"sv) {
+        return SkBlender::Mode(SkBlendMode::kDstOver);
+    }
+    if (current_compositing_and_blending_operator == "source-in"sv) {
+        return SkBlender::Mode(SkBlendMode::kSrcIn);
+    }
+    if (current_compositing_and_blending_operator == "destination-in"sv) {
+        return SkBlender::Mode(SkBlendMode::kDstIn);
+    }
+    if (current_compositing_and_blending_operator == "source-out"sv) {
+        return SkBlender::Mode(SkBlendMode::kSrcOut);
+    }
+    if (current_compositing_and_blending_operator == "destination-out"sv) {
+        return SkBlender::Mode(SkBlendMode::kDstOut);
+    }
+    if (current_compositing_and_blending_operator == "source-atop"sv) {
+        return SkBlender::Mode(SkBlendMode::kSrcATop);
+    }
+    if (current_compositing_and_blending_operator == "destination-atop"sv) {
+        return SkBlender::Mode(SkBlendMode::kDstATop);
+    }
+    if (current_compositing_and_blending_operator == "xor"sv) {
+        return SkBlender::Mode(SkBlendMode::kXor);
+    }
+    if (current_compositing_and_blending_operator == "lighter"sv) {
+        return SkBlender::Mode(SkBlendMode::kPlus);
+    }
+    if (current_compositing_and_blending_operator == "plus-darker"sv) {
+        return SkRuntimeEffect::MakeForBlender(SkString(R"(
+            vec4 main(vec4 source, vec4 destination) {
+                return max(vec4(0.0), source + destination - vec4(1.0));
```

Comment:
how does this behave?
```suggestion
                return max(vec4(0.0), vec4(1.0) - ((vec4(1.0) - destination) + (vec4(1.0) - source))));
```

Reply:
Sadly, this does not work either...

I failed to find an implementation of this mode using Skia on Github. Even WebKit, the only Engine that supports it, has no implementation using Skia. I think the problem is the transparent black background, but I currently have no workaround.